js/about/safebrowsing.js

```diff
@@ -22,7 +22,7 @@ class SafebrowsingPage extends React.Component {
   }
 
   render () {
-    return <div className='errorContent' id='safebrowsingPage' style={{height: 'auto'}}>
```

Comment:
I was not sure what was the style for. If requred for some reason I will revert this.